app/src/main/java/openfoodfacts/github/scrachx/openfood/views/MainActivity.java

```diff
@@ -257,7 +257,11 @@ public void onDrawerSlide(View drawerView, float slideOffset) {
                             if (otherOFAppInstalled) {
                                 Intent LaunchIntent = getPackageManager()
                                         .getLaunchIntentForPackage(BuildConfig.OFOTHERLINKAPP);
-                                startActivity(LaunchIntent);
+                                if (LaunchIntent != null) {
+                                    startActivity(LaunchIntent);
+                                }else{
+                                    Toast.makeText(this, "App Disabled", Toast.LENGTH_SHORT).show();
```

Comment:
This needs to be a string resource
> Application is not installed

Reply:
Is that possible to propose reenabling the app ?

Reply:
Ok. I can write a function to alert the user that the app i currently disabled and suggest to take the user to the disabled app's setting to re-enable it.
What do you think @teolemon ?